source/deployment/mobile-app-deployment.rst

```diff
@@ -0,0 +1,571 @@
+
+For now, focus on that last part, “... your apps are simpler to configure, secure and deploy.” AppConfig provides the most efficient and scalable path for success. As an admin this means easier deployment and management of mobile applications. And again, when it comes to Mattermost, it’s our only supported approach.
+
+Application (App) Wrapping
+--------------------------
+From the article we `referenced earlier <https://www.computerworld.com/article/3209907/app-wrapping-the-key-to-more-secure-mobile-app-management.html>`_, application(app) wrapping involves “...the use of an SDK from an app or EMM vendor that allows a developer or admin to deploy an API that enables management policies to be set up.” 
+
+When going this route, there are two options:
+
+**Option 1**
+    The EMM provider gives you their libraries, and then you go to the source code for the app. Using the libraries you "wrap" the source and repackage the application. This approach will take significant development time and associated frustration. In the end you should have an app that is now wrapped with the EMM libraries. The hope, but not the guarantee, is that you have an app with an additional layer. This allows you to manage and secure the app on a user’s mobile device.
+
+**Option 2**
+    The EMM provider gives you a tool used for wrapping the app. In more modern cases, it’s a simple checkbox in your EMM application when configuring the app. These tools then inject all the needed code to wrap it. Then the tool (or EMM Provider) builds it back up. This allows you to distribute the new, wrapped app. The wrapped app has a layer allowing you to manage and secure the app on a user’s mobile device. 
+
+Until recently app wrapping has been the common approach. It does not come without risks and challenges to scalability though. 
+
+For example, most modern applications follow continuous development. Each time an application changes it will need to go through the process described above. 
+
+There is also functionality and compatibility risk. This is a known issue for the Mattermost application and app wrapping tools.
+
+This incompatibility is not an issue with the Mattermost application. It results from the proprietary nature of the provider's tools. To make matters worse, there is no course of action to address compatibility issues.
+
+In the end, this sets app wrapping in a negative light. This is the reason VMWare and other EMM providers came together to create a standard. AppConfig is a modern, efficient, and scalable approach to enterprise mobile management. In that regard, the Mattermost application benefits from being built around this standard.
```

Comment:
Why the special mention to VMWare ?

Reply:
Unintentional. Probably just an artifact of writing. We can add names from the list at appconfig.org or remove this reference. 0/5

Reply:
+1. The day of writing I think the site was down and I never went back to add it.

Reply:
Suggestion: `This is the reason the [AppConfig Community](https://www.appconfig.org) came together to create a standard.`

Reply:
+1